lib/src/config/route/app_routes.dart

```diff
@@ -16,6 +24,28 @@ class AppRoutes {
         return MaterialPageRoute(
           builder: (_) => const HomePage(),
         );
+      case RoutesConstants.detailRoute:
+        return MaterialPageRoute(
+          builder: (_) => DetailPage(
+            id: (settings.arguments! as Map<String, dynamic>)[detailPageId]
+                as int,
+            title: (settings.arguments!
+                as Map<String, dynamic>)[detailPageTitle] as String,
+            aggregateLikes: (settings.arguments!
+                as Map<String, dynamic>)[detailPageAggregateLikes] as int,
+            instructions: (settings.arguments!
+                as Map<String, dynamic>)[detailPageInstructions] as String,
+            image: (settings.arguments!
+                as Map<String, dynamic>)[detailPageImage] as String,
+            storage: GetStorageLocal.instance,
```

Comment:
Instead to send a lot of diferrent parameter into detail page, could be better approach to crear an object that contains (id, title, agregateLies, instructions)